### DISCUSSION TITLE
There should be an option to fail fast

### DISCUSSION THREAD

**Discussion by sha237gh** [vercel/next.js#49469] [Ideas]
### Goals

Since a single failed page render will fail the whole build there should be an option to exit the build process on the first error because builds can take a very long time to complete.

### Background

Currently nextjs doesn't offer any settings for building and it should offer devs flexibility when it comes to this time consuming step.

### Proposal

e.g. as argument to the build command: `next build --fail-fast`

**Comment by remisture** [comment]
Yes, please 👍
